tests/rptest/tests/partition_move_interruption_test.py

```diff
@@ -48,17 +48,74 @@ def __init__(self, ctx, *args, **kwargs):
         self.partition_count = 20
         self.consumer_timeout_seconds = 90
 
+    def get_moving_to_node_metrics(self, node):
+        metrics = self.redpanda.metrics_sample(
+            "moving_to_node", [node],
+            metrics_endpoint=MetricsEndpoint.PUBLIC_METRICS)
+        return int(sum([metric.value for metric in metrics.samples]))
+
+    def get_moving_from_node_metrics(self, node):
+        metrics = self.redpanda.metrics_sample(
+            "moving_from_node", [node],
+            metrics_endpoint=MetricsEndpoint.PUBLIC_METRICS)
+        return int(sum([metric.value for metric in metrics.samples]))
+
+    def get_cancelled_movements_for_node_metrics(self, node):
+        metrics = self.redpanda.metrics_sample(
+            "node_cancelled_movements", [node],
+            metrics_endpoint=MetricsEndpoint.PUBLIC_METRICS)
+        return int(sum([metric.value for metric in metrics.samples]))
+
+    def metrics_correct(self, prev_assignment, assignmets):
+        prev_assignment_nodes = [a["node_id"] for a in prev_assignment]
+        assignment_nodes = [a["node_id"] for a in assignmets]
+        moving_from_node_sum = 0
+        moving_to_node_sum = 0
+        result = True
```

Comment:
can we just use early returns? looks like result is just `and`-ed from several checks